PGIP/Output.hs

```diff
@@ -0,0 +1,6 @@
+module PGIP.Output
```

Comment:
I'm no friend of re-exports. In a long list of imports two direct imports will not harm.